cups/translate-attrs.go

```diff
@@ -67,6 +67,16 @@ func getUUID(printerTags map[string][]string) string {
 }
 
 func getState(printerTags map[string][]string) cdd.CloudDeviceStateType {
+	// offline-report reason is set when printer is offline (e.g. USB cable is disconnected)
```

Comment:
CUPS doesn't detect anything when I unplug my USB printer. So I can't do any kind of diff here.

Can you run `ipptool -t -v ipp://localhost:631/printers/PRINTER /usr/share/cups/ipptool/get-printer-attributes.test` while connected and disconnected, then diff the results?